### PR TITLE
Initial commit and PR

### PR DESCRIPTION
**Pull request by workingjubilee**
TODO:
- 10 event listeners
- nest 2 similar event listeners and stop event Propagation
- stop nav from refreshing the page using preventDefault()
- stretch?

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -335,9 +361,9 @@ img {
   align-items: center;
   width: 200px;
   height: 45px;
-  background-color: #17A2B8;
```
Comment (GannonDetroit) [inline css/index.css:338]: I can't imagine you lower cased all these by hand, what code extension are you using that does this??

File: js/index.js [hunk 2]
```diff
@@ -1 +1,192 @@
+
+
+// my tools are
```
Comment (GannonDetroit) [inline js/index.js:3]: good job bringing this in as  comment to make referencing it faster/easier on yourself.

File: less/animations.less [hunk 3]
```diff
@@ -0,0 +1,30 @@
+@-webkit-keyframes pulse {
```
Comment (GannonDetroit) [inline less/animations.less:1]: awesome job with the animations

File: less/mixins.less [hunk 4]
```diff
@@ -10,6 +10,20 @@
           border-radius: @radius;
 }
 
+.pulse {
```
Comment (GannonDetroit) [inline less/mixins.less:13]: I love and you went back and added a new mixin. Way to go and find ways to re-apply previous lessons. THIS is how you stay sharp and on top of things.

**Review by GannonDetroit — COMMENTED** [review]
Fantastic job on this and on the 10 events you chose to pick. Top-notch and I'll merge the branch

File: css/index.css (hunk 1 above)
Comment (workingjubilee) [reply css/index.css:338]: So the funny thing is, when I look at the variables.less file, it's the same, and I know I didn't hand-touch the index.css, so I believe this is the consequence of my OS being Linux, which is ultra-picky about casing, and running less-watch-compiler. Something in the code is probably recompiling the CSS with a very specific idea about whether it should be upper case or lower case.

**Pull request merged**